vision_agent/agent/vision_agent.py

```diff
@@ -407,8 +407,6 @@ def chat_with_code(
             code_interpreter.download_file(
                 str(remote_artifacts_path.name), str(self.local_artifacts_path)
             )
-            artifacts.load(self.local_artifacts_path)
-            artifacts.save()
```

Comment:
not sure if this is intended. but I'm not too familiar with the code.

Reply:
This was causing front end some issues. I did some local tests and I don't think removing it causes any issues